app/src/main/java/eu/kanade/tachiyomi/ui/player/PlayerActivity.kt

```diff
@@ -299,6 +299,7 @@ class PlayerActivity :
         setVisibilities()
 
         playerControls.showAndFadeControls()
+        playerControls.hideControls(true)
```

Comment:
```suggestion
```

maybe instead of showing and then immediately hiding, try not showing the controls in the first place